### ISSUE TITLE
Support llama3.2-vision?

### ISSUE THREAD

**Issue by ZubikIT** [ArdaGnsrn/ollama-php#5]
Hi, how use model llama3.2-vision?
https://ollama.com/blog/llama3.2-vision

**Comment by zachary** [comment]
yes, I tested it successfully, put array [$base64] into index 'images' under index 'messages', that is all
